export const showErrorAlert = () => {
    alert('오류가 발생했습니다. 다시 시도해주세요.');
};
